gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -137,6 +136,8 @@ def __enter__(self):
     self.dataset_id = 'integration_tests_{}'.format(
         datetime.now().strftime('%Y%m%d_%H%M%S'))
     dataset_ref = client.dataset(self.dataset_id)
+    # TODO(bashir): This does not seem to be right, the client argument is
```

Comment:
I haven't run this myself yet, but looks like this has the same structure as https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/bigquery/cloud-client/quickstart.py ?

Reply:
Updated the TODO per our offline discussion. For the record, the issue is the incompatibility between different versions of bigquery API.